app/webpacker/controllers/trixeditor_controller.js

```diff
@@ -3,10 +3,43 @@ import { Controller } from "stimulus";
 export default class extends Controller {
   connect() {
     window.addEventListener("trix-change", this.#trixChange);
+    this.#trixInitialize();
+    window.addEventListener("trix-initialize", this.#trixInitialize);
```

Comment:
Do we need both line here ? You would think the listener on `trix-inithialize` would be enough ?

Reply:
I thought so, and that's the normal way of doing it. Unfortunately, for some reasons, `trix-initialize` can be triggered _before_ the stimulus controller is connected to the trixeditor. But, sometimes, that's the opposite. That's why I've added the two ways....

Reply:
Tricky one, I can't think of any other solution :man_shrugging: